spec/player_spec.rb

```diff
@@ -70,10 +78,30 @@
       expect(@player.highest_word_score).to be_nil
     end
     it "returns the highest scoring word score" do
-      @player.play("cat")
-      @player.play("crazy")
-      @player.play("orange")
+      @player.create_mock({plays: %w(CAT CRAZY ORANGE)})
       expect(@player.highest_word_score).to eq 19
     end
   end
+  describe "#tiles" do
+    it "returns an empty array of tiles at the beginning of the game" do
+      expect(@player.tiles).to eq []
+    end
+  end
+  describe "#draw_tiles(tile_bag)" do
+    before :each do
+      @tilebag = Scrabble::TileBag.new
+    end
+    it "draws 7 tiles" do
+      expect(@player.draw_tiles(@tilebag).length).to eq 7
+    end
+    it "adds 7 tiles to the players tiles array" do
+      @player.draw_tiles(@tilebag)
+      expect(@player.tiles.length).to eq 7
+    end
+    it "adds 3 tiles to the players tiles array already containing 4 tiles" do
+      @player.create_mock({tiles: %w(W O R D)})
+      @player.draw_tiles(@tilebag)
+      expect(@player.tiles.length).to eq 7
```

Comment:
It might be a good idea to also expect `@tilebag.remaining_tiles` to have 3 fewer tiles than at the beginning of this test.